avocado/plugins/yaml_to_mux.py

```diff
@@ -45,6 +46,23 @@
 __RE_FILE_SUBS = re.compile(r'(?<!\\)\\:')  # substitute '\\:' but not '\\\\:'
 
 
+class _BaseLoader(Loader):
+    """
+    YAML loader with additional features related to mux
+    """
+    pass
+
+
+_BaseLoader.add_constructor(u'!include', lambda loader,
```

Comment:
I don't see the reason why you further manipulate the `_BaseLoader` class outside its definition scope.  Is there a particular reason?  If not, something like this:

```diff
diff --git a/avocado/plugins/yaml_to_mux.py b/avocado/plugins/yaml_to_mux.py
index 2b1d73d..0e05f37 100644
--- a/avocado/plugins/yaml_to_mux.py
+++ b/avocado/plugins/yaml_to_mux.py
@@ -50,17 +50,15 @@ class _BaseLoader(Loader):
     """
     YAML loader with additional features related to mux
     """
-    pass
 
-
-_BaseLoader.add_constructor(u'!include', lambda loader,
-                            node: mux.Control(YAML_INCLUDE))
-_BaseLoader.add_constructor(u'!using',
-                            lambda loader, node: mux.Control(YAML_USING))
-_BaseLoader.add_constructor(u'!remove_node',
-                            lambda loader, node: mux.Control(YAML_REMOVE_NODE))
-_BaseLoader.add_constructor(u'!remove_value',
-                            lambda loader, node: mux.Control(YAML_REMOVE_VALUE))
+    Loader.add_constructor(u'!include', lambda loader,
+                           node: mux.Control(YAML_INCLUDE))
+    Loader.add_constructor(u'!using',
+                           lambda loader, node: mux.Control(YAML_USING))
+    Loader.add_constructor(u'!remove_node',
+                           lambda loader, node: mux.Control(YAML_REMOVE_NODE))
+    Loader.add_constructor(u'!remove_value',
+                           lambda loader, node: mux.Control(YAML_REMOVE_VALUE))
 
 
 class Value(tuple):     # Few methods pylint: disable=R0903
```

Would be better.